Runtime/Core/AutoUnwrapSettings.cs

```diff
@@ -20,69 +23,70 @@ public static AutoUnwrapSettings defaultAutoUnwrapSettings
+            /// The right top bounding point of the projected UVs is aligned with UV coordinate {1, 1}.
             /// </summary>
             UpperRight,
             /// <summary>
-            /// The middle left bound of the projected UVs is aligned with UV coordinate {0, .5}.
+            /// The middle left bounding point of the projected UVs is aligned with UV coordinate {0, .5}.
             /// </summary>
             MiddleLeft,
             /// <summary>
             /// The center bounding point of the projected UVs is aligned with UV coordinate {.5, .5}.
             /// </summary>
             MiddleCenter,
             /// <summary>
-            /// The middle right bound of the projected UVs is aligned with UV coordinate {1, .5}.
+            /// The middle right bounding point of the projected UVs is aligned with UV coordinate {1, .5}.
             /// </summary>
             MiddleRight,
             /// <summary>
-            /// The lower left bound of the projected UVs is aligned with UV coordinate {0, 0}.
+            /// The lower left bounding point of the projected UVs is aligned with UV coordinate {0, 0}.
             /// </summary>
             LowerLeft,
             /// <summary>
-            /// The lower center bound of the projected UVs is aligned with UV coordinate {.5, 0}.
+            /// The lower center bounding point of the projected UVs is aligned with UV coordinate {.5, 0}.
             /// </summary>
             LowerCenter,
             /// <summary>
-            /// The lower right bound of the projected UVs is aligned with UV coordinate {1, 0}.
+            /// The lower right bounding point of the projected UVs is aligned with UV coordinate {1, 0}.
             /// </summary>
             LowerRight,
             /// <summary>
-            /// UVs are not aligned following projection.
+            /// UVs are not aligned with any projection.
             /// </summary>
             None
         }
 
         /// <summary>
-        /// Describes how the projected UV bounds are optionally stretched to fill normalized coordinate space.
+        /// Describes how ProBuilder optionally stretches the projected UV bounds to fill the normalized coordinate space.
         /// </summary>
         public enum Fill
         {
             /// <summary>
-            /// UV bounds are resized to fit within a 1 unit square while retaining original aspect ratio.
+            /// Retain original aspect ratio while resizing UV bounds to fit within a 1 unit square.
             /// </summary>
             Fit,
             /// <summary>
-            /// UV bounds are not resized.
+            /// Don't resize UV bounds.
             /// </summary>
             Tile,
             /// <summary>
-            /// UV bounds are resized to fit within a 1 unit square, not retaining aspect ratio.
+            /// Don't retain aspect ratio while resizing UV bounds to fit within a 1 unit square.
             /// </summary>
```

Comment:
Yes, because enum values describes what the value tells Unity to do, so it's using an imperative for. The main entry for the enum type and all other members always use a 3rd-person description. Details are on this Confluence page: https://confluence.unity3d.com/display/DOCS/API+docs+templates#heading-Enums

Reply:
Ok nice to know :) Thanks